integration/tests/cook/util.py

```diff
@@ -42,9 +43,41 @@ def has_docker_service():
 _default_admin_name = 'root'
 _default_impersonator_name = 'poser'
 
+
 def _get_default_user_name():
     return os.getenv('USER', _default_user_name)
 
+
+@functools.lru_cache()
+def _test_user_ids():
+    """
+    Get the numeric user suffixes for this test worker.
+    Returns the range 0 to 1 million if COOK_MAX_TEST_USERS is not set.
+    If this is a distributed run with a limited number of users,
+    e.g., 10 per worker, then this function returns range(0, 10) for worker 0,
+    or range(20, 30) for worker 2.
+    """
+    pytest_worker = os.getenv('PYTEST_XDIST_WORKER')
```

Comment:
What sets this environment variable? Might be nice to add a comment

Reply:
The pytest-xdist (parallel testing) plugin sets this for each of its worker sub-processes.